kythe/java/com/google/devtools/kythe/platform/kzip/KZipWriter.java

```diff
@@ -31,6 +33,9 @@
   private final ZipOutputStream output;
   private final Gson gson;
 
+  // We may
```

Comment:
maybe fill out the comment

// We store our own set of paths written, analogous to ZipOutputWriter.names, so that we avoid
// raising exceptions by writing duplicates.

Reply:
Done